crates/re_build_info/src/build_info.rs

```diff
@@ -166,8 +166,8 @@ fn crate_version_from_build_info_string() {
 
     {
         let expected_crate_version = build_info.version;
-        let crate_version = CrateVersion::try_parse_from_build_info_string(build_info_str).unwrap();
+        let crate_version = CrateVersion::try_parse_from_build_info_string(build_info_str);
 
-        assert_eq!(expected_crate_version, crate_version);
+        assert_eq!(Ok(expected_crate_version), crate_version);
```

Comment:
nit: isn't this caught by the exception for allowing unwraps in test?

Reply:
Yes. I just find this nicer/more declarative ;) ...and the log will be more descriptive: instead of panic you will see the actual values.

Reply:
makes sense (: